### PR TITLE
service: update operatorkit (new Resource)

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: service/persistentvolume/framework.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+	gsclient "github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/informer"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v1"
```
Comment (rossf7) [inline service/persistentvolume/framework.go]: This import looks wrong.

Comment (kopiczko) [reply service/persistentvolume/framework.go]: Thank you <3

Comment (kopiczko) [reply service/persistentvolume/framework.go]: `dep ensure` removed that garbage. Thanks you saved me from many struggles :)

**Comment by kopiczko** [comment]
CI cries about `Error response from daemon: manifest for quay.io/giantswarm/golang:1.10.0 not found`

**Comment by corest** [comment]
Probably related to the architect update, no?

**Comment by xh3b4sd** [comment]
Lets talk here: https://gigantic.slack.com/archives/C3C7ZQXC1/p1520849724000183.

**Comment by kopiczko** [comment]
https://github.com/giantswarm/retagger/pull/60

**Comment by kopiczko** [comment]
@corest is there a way to test that? Is it safe to merge like that?

**Comment by corest** [comment]
@kopiczko I'll test it on minikube to verify

**Comment by kopiczko** [comment]
@corest have you had a chance to do so?

**Comment by corest** [comment]
@kopiczko tested and fixed some issues there, can be merged now

**Comment by kopiczko** [comment]
Thank you!

**Pull request merged**